packages/argo-run/src/webpack-config.ts

```diff
@@ -84,8 +84,7 @@ export function createWebpackConfiguration({
     module: {
       rules: [
         {
-          test: /\.js$/,
-          exclude: /node_modules/,
+          test: /\.(jsx?|esnext)$/,
```

Comment:
I'd prefer we keep the rule for handling `.js` in `node_modules` separate from the one for app code, most likely other parts of it will be different (at least the cacheing, possibly also the Babel options at some point).

Reply:
Changed it so that this is only for app code and https://github.com/Shopify/argo-checkout/pull/57/files#diff-2087c6e948a1e8019253ff0fdfbd74b4e3911a6a6e5eb34fb82ca6000b0707b0R103 for node_modules